_data/regions.yml

```diff
@@ -20,4 +29,4 @@
   name: United Kingdom
 
 - id: us
-  name: United States
+  name: United States
```

Comment:
Any reason for the newline being removed?

Reply:
IDE Magic
![](https://www.awesomeinventions.com/wp-content/uploads/2019/04/imagination-spongebob-meme.jpg)